HW6/YaRa1312/Pleshyvtseva PythonCore HW 6.3.py

```diff
@@ -0,0 +1,15 @@
+# 6.3
+
+def characters_number_calculation(user_string):
+    '''
+    The function calculates characters in the string
+    entered by a user and give the quality of usage of each
+    character in a dict where {character: quality of usage}
+    '''
```

Comment:
another variant:
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result